src/applications/facility-locator/tests/e2e/gaEvents.cypress.spec.js

```diff
@@ -16,8 +16,8 @@ describe('Google Analytics FL Events', () => {
       cy.get('#street-city-state-zip').type('Austin, TX');
       cy.get('#facility-type-dropdown').select('VA health');
       cy.get('#facility-search').click();
-      cy.injectAxe();
-      cy.axeCheck();
+      // cy.injectAxe();
+      // cy.axeCheck();
```

Comment:
I don't know which is more desirable, but you may [pass a `skkipFailures` option](https://github.com/avanslaars/cypress-axe#skipfailures-optional-defaults-to-false) to continue running aXe checks without interrupting the build but still output failures in the console.